application/locales/global.en.yaml

```diff
@@ -223,7 +223,7 @@
 
 # New Message notification subject
 - id: Email.Subject.Message.Created
-  translation: New message from {{.sentByNickname}} on {{.AppName}}
+  translation: {{.AppName}} message from {{.sentByNickname}} about {{.postTitle}}
```

Comment:
I guess this broke some tests.

Reply:
I guess you can't have a leading `{` in a string.